recipes-domu/domu-image-weston/files/meta-xt-prod-extra/recipes-kernel/kernel-module-gles/kernel-module-gles_1.10.bbappend

```diff
@@ -1,5 +1,5 @@
 require inc/xt_shared_env.inc
 
 PVRKM_URL = "git://git@gitpct.epam.com/epmd-aepr/pvr_km_vgpu_img.git"
-BRANCH = "1.10/ED5187610"
+BRANCH = "1.10/ED5187610_GCC-8"
```

Comment:
I am wondering can we avoid introducing a separate branch for just keeping the warning off? Can we keep appropriate patch in meta-layer or just apply required patch to the existing 1.10/ED5187610 branch?